lace/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lace"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Not a concern, just for my own view: it looks like we have some new options in some binaries so it makes sense to bump a major(-ish) version to me